### ISSUE TITLE
CRAN timeline?

### ISSUE THREAD

**Issue by vnijs** [rstudio/rstudioapi#92]
I was wondering if there is a timeline for when `rstudioapi` might be updated on CRAN? The reason I ask is that there have been a few fixes in rstudioapi related to the filebrowser (e.g., https://github.com/rstudio/rstudioapi/issues/58) that I rely on in a package I would like to update on CRAN.

**Comment by kevinushey** [comment]
We don't have a timeline yet, but we'll likely aim for a release around the time that we try to get v1.2 stabilized (hopefully starting this summer).

**Comment by jjallaire** [comment]
It seems like we could certainly do a release prior to then if there isn't a reason we are waiting on v1.2 -- @kevinushey are there any special reasons for us to wait?

**Comment by kevinushey** [comment]
I don't think so but we might want to do some issue triage to see if there's anything we want to fix before a CRAN submission.

**Comment by vnijs** [comment]
That would be great! I have posted a few issues (see below) but the main one for me, by far, is #91. I think I've been able to integrate a shiny app to work nicely with Rstudio projects but a full (navbarPage) shiny app may be too big to conveniently use Viewer. Unfortunately, having the main Rstudio window show in front of an Rstudio Window (at least on Windows) when using `selectFile` makes this impractical as well. Not sure if this should be posted to rstudio/rstudio though as it seems related to (https://github.com/rstudio/rstudio/issues/2566) and perhaps (https://github.com/rstudio/rstudio/issues/2516)

Issue relate to `selectFiles`: https://github.com/rstudio/rstudioapi/issues/91, https://github.com/rstudio/rstudioapi/issues/80, https://github.com/rstudio/rstudioapi/issues/79, https://github.com/rstudio/rstudioapi/issues/61

In case it is relevant, the code for the app I'm referring to is available @ https://github.com/radiant-rstats/radiant.data. You can also install it using:

`install.packages("radiant.data", repos = "https://radiant-rstats.github.io/minicran/")`

**Comment by vnijs** [comment]
FYI Just noticed that on Linux (and Rstudio server pro) the file browser opened by `selectFile` stays *behind* the Rstudio Window running the shiny app
